### PR TITLE
android 13 port

### PR DESCRIPTION
**Pull request by empratyush**

### CODE REVIEW

File: framework/java/android/net/wifi/WifiConfiguration.java [hunk 1]
```diff
@@ -1811,16 +1812,23 @@ public boolean isOpenNetwork() {
      */
     public static final int RANDOMIZATION_AUTO = 3;
 
+    /**
+     * @hide
```
Comment (muhomorr) [inline framework/java/android/net/wifi/WifiConfiguration.java]: Remove this line, make it the same as other constants.

File: framework/api/system-current.txt [hunk 2]
```diff
@@ -409,6 +409,7 @@ package android.net.wifi {
     field @Deprecated public static final int METERED_OVERRIDE_METERED = 1; // 0x1
     field @Deprecated public static final int METERED_OVERRIDE_NONE = 0; // 0x0
     field @Deprecated public static final int METERED_OVERRIDE_NOT_METERED = 2; // 0x2
+    field @Deprecated public static final int RANDOMIZATION_ALWAYS = 100; // 0x64
```
Comment (muhomorr) [inline framework/api/system-current.txt]: Remove, not needed and will break the build.

File: framework/java/android/net/wifi/WifiConfiguration.java (hunk 1 above)
Comment (empratyush) [reply framework/java/android/net/wifi/WifiConfiguration.java]: okay,

**Comment by muhomorr** [comment]
`RANDOMIZATION_ALWAYS` needs to be added to framework/api/current.txt

**Pull request merged**